jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -96,6 +98,34 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    override fun setupDtlsTransport() {
+        dtlsTransport.incomingDataHandler = object : DtlsTransport.IncomingDataHandler {
+            override fun dtlsAppDataReceived(buf: ByteArray, off: Int, len: Int) {
+                this@EndpointK.dtlsAppPacketReceived(buf, off, len)
+            }
+        }
+        dtlsTransport.outgoingDataHandler = object : DtlsTransport.OutgoingDataHandler {
+            override fun sendData(buf: ByteArray, off: Int, len: Int) {
+                iceTransport.send(buf, off, len)
+            }
+        }
```

Comment:
Hm, I'm surprised the Kotlin syntax for these two handlers is so much heavier than the Java syntax, that's the opposite of what I'd usually expect.  There's not simpler Kotlin syntax for this?

Reply:
SAM conversions for kotlin interfaces isn't in until 1.4 :/

Reply:
We could define a typealias for the handler instead of an interface, but these are annoying because they can't be scoped.

Reply:
Another reason to get around to the 1.4 update...